strings/tips/meta.txt

```diff
@@ -1,39 +1,21 @@
 Dying is part of the game.
-Almost every specialist has been reworked.
+While holding nothing on your active hand, click a person on fire with help intent to put the fire out. This applies both xenos and humans.
 You can select locations on the doll by using the number pad.
-As a human, say '*medic' to call out for medical attention.
-As a human, melee a person on fire with help intent to put the fire out.
-As a human, use 'resist' to stop drop and roll when on fire.
-Not being ready at the start of the round means xenos get more burrowed larva, and that you don't get to play as quickly.
+While you have your throw mode active, you can automatically catch things that are thrown at you, if you are able to hold the item as a species.
+Remember: Fore is North, Aft is South, Starboard is East, and Port is West.
```

Comment:
I feel that this one isn't the best, ship maps are orientated differently.

Reply:
Ah, I checked the Pillar of Spring and it's the case, so... yeah, maybe it's for the best.